cmd/containerd-shim-runhcs-v1/delete.go

```diff
@@ -18,6 +17,29 @@ import (
 	"go.opencensus.io/trace"
 )
 
+// LimitedRead reads at max `readLimitBytes` bytes from the file at path `filePath`. If the file has
+// more than `readLimitBytes` bytes of data then first `readLimitBytes` will be returned.
+func limitedRead(filePath string, readLimitBytes int64) ([]byte, error) {
+	f, err := os.Open(filePath)
+	if err != nil {
+		return nil, errors.New(fmt.Sprintf("limited read failed to open file: %s", err))
```

Comment:
I'd say to just wrap the error, we should probably add the file name in the error string also. 
```go
return nil, errors.Wrapf(err, "limited read failed to open file %q", filePath)
```

Reply:
Same for line 35 and 39

Reply:
errors pacakge that we are using here is from golang errors which doesn't have `Wrap/Wrapf` methods. I just used the existing package but I can replace that with `github.com/pkg/errors` if needed.

Reply:
Yea it's harmless to swap them, it seems like for new code we generally wrap now instead of doing the `fmt.Errorf` dance